From 85c454ca68b95ea94341448b0257bbb54d960347 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 14 Aug 2018 17:00:09 +0100
Subject: [PATCH 73/74] jni: Resolve unused and uninitialized variable errors

 * Required for devices that set QCOM_NO_FM_FIRMWARE to true.

Change-Id: Ie940ba3e0736197b776d86718077e2b220d9ea5c
---
 jni/android_hardware_fm.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/jni/android_hardware_fm.cpp b/jni/android_hardware_fm.cpp
index 9cd8187..d0f870a 100644
--- a/jni/android_hardware_fm.cpp
+++ b/jni/android_hardware_fm.cpp
@@ -649,7 +649,7 @@ static jint android_hardware_fmradio_FmReceiverJNI_acquireFdNative
         (JNIEnv* env, jobject thiz __unused, jstring path)
 {
     int fd;
-    int i, err;
+    int i = 0, err;
     char value[PROPERTY_VALUE_MAX] = {'\0'};
     char versionStr[40] = {'\0'};
     int init_success = 0;
@@ -1204,7 +1204,9 @@ static jint android_hardware_fmradio_FmReceiverJNI_setNotchFilterNative
     jboolean aValue)
 {
     char value[PROPERTY_VALUE_MAX] = {'\0'};
+#ifndef QCOM_NO_FM_FIRMWARE
     int init_success = 0,i;
+#endif
     char notch[PROPERTY_VALUE_MAX] = {0x00};
     int band;
     int err = 0;
-- 
2.17.1

